### ISSUE TITLE
Multi-Import Layout inherits all the interfaces in flatten file

### ISSUE THREAD

**Issue by naszam** [makerdao/dss-interfaces#69]
## To reproduce:

- `dapp install https://github.com/makerdao/dss-interfaces`

```solidity
// SPDX-License-Identifier: AGPL-3.0-or-later
pragma solidity 0.6.12;

import { VatAbstract, VowAbstract } from "dss-interfaces/Interfaces.sol";

contract TestDssInterfaces {
}
```

- `dapp --use solc:0.6.12 build`
- `hevm flatten --source-file "src/TestDssInterfaces.sol" > out/flat.sol`

## Output
- [TestDssInterfaces Flatten Output](https://hackmd.io/@naszam/rkySGhJwq)

**Comment by brianmcmichael** [comment]
This is by design. You can avoid it by importing individual interfaces.

**Comment by naszam** [comment]
Yes, this was addressed in #71 and included in the checklists as well, to avoid multi-import layout (see [here](https://github.com/makerdao/pe-checklists)).
Closing for now.
